docker/amazonlinux/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM amazonlinux:2 as rust
+FROM amazonlinux:2023 as rust
```

Comment:
are you sure ? - 2023 is fedora - aka dnf package manager - vs :2 which is still RHEL 7 based

Reply:
rolling back